README.md

```diff
@@ -198,36 +198,10 @@ automatically.
 | int         | sys-map.int.bgdi.ch  | /                      | master         |
 | prod        | sys-map.prod.bgdi.ch | /                      | master         |
 
-On the `dev` and `int` targets, deployement is done **automatically** via the [CI for web-mapviewer](https://github.com/geoadmin/infra-terraform-bgdi-builder/tree/master/projects/web_mapviewer#ci-for-web-mapviewer).
+On the `dev` and `int` targets, deployment is done **automatically** via the [CI for web-mapviewer](https://github.com/geoadmin/infra-terraform-bgdi-builder/tree/master/projects/web_mapviewer#ci-for-web-mapviewer).
 
 A [test link](https://github.com/geoadmin/web-mapviewer/blob/bug_update_doc_regarding_deploy/.github/workflows/add-testlink-to-pr.yml) is also added to the description of every PR.
 
-### Manual deploy
-
-A bash script [deploy.sh](https://github.com/geoadmin/infra-terraform-bgdi-builder/blob/master/projects/web_mapviewer/scripts/deploy.sh)
-is used for manual deploy, either from a local directory or a bucket from the CI.
-
-    ./scripts/deploy.sh: --staging STAGING {--version VERSION | --local-src DIR} [--preview TEST_LINK]
-
-    Deploy web-mapviewer on the given staging. Either deploy a version from the
-    build-artifacts-swisstopo bucket (with --version option), or a local build version
-    using the --local-src DIR option.
-
-    OPTIONS:
-      -h|--help               Print the help and exit.
-      -s|--staging STAGING    Staging to deploy; dev|int|prod. Default; dev
-      -v|--version VERSION    Version to deploy.
-
-On `prod`, check [deploy on prod](https://github.com/geoadmin/infra-terraform-bgdi-builder/tree/master/projects/web_mapviewer#deploy-on-prod) and use the script from within `infra-terraform-bgdi-builder/projects/web_mapviewer` to deploy **manually**.
-
-> **_NOTE:_**  
-> If deploying manually to `prod`, wait until the CI has finished building the project, as the deploy script only copy files.
-
-Depending on the target (`dev|int|prod`), you will have to build and bundle/minify the app (for `int` and `prod`) or simply build the app without minification (for `dev`) prior to deplay (`npm run build:(dev|int|prod)`)
-
-- Only `develop` branch can be deployed at the root of the `dev` bucket.
-- Only `master` branch can be deployed at the root of `int` and `prod` buckets.
```

Comment:
Why removing this ? This is still actual if we want/need to do manual deploy.

Reply:
because the deploy script has been removed by https://github.com/geoadmin/web-mapviewer/pull/293

Reply:
ho right, I thought it was talking only about this script, but this is still relevant in some cases... my bad